src/Resources/views/storefront/layout/scroll-up.html.twig

```diff
@@ -0,0 +1,11 @@
+{% sw_extends '@Storefront/storefront/layout/scroll-up.html.twig' %}
+
+{% block layout_scroll_up_button %}
+    <div class="scroll-up-button js-scroll-up-button mb-5">
```

Comment:
so adding the bootstrap mb-5 css classname is the change avoiding scroll-up button overlapping with Zendesk Widget ?

Reply:
yes, it's bootstrap class responsible for margin bottom: https://getbootstrap.com/docs/4.4/utilities/spacing/